lib/bundler/cli.rb

```diff
@@ -151,6 +151,8 @@ def check
         Bundler.rubygems.security_policy_keys.join('|')
     method_option "without", :type => :array, :banner =>
       "Exclude gems that are part of the specified named group."
+    method_option "with", :type => :array, :banner =>
+      "Include gems that are part of the specified named group."
```

Comment:
I did most reasoning in https://github.com/bundler/bundler-features/issues/68, which is why I spared it in the PRs description :)